lib/browser/api/browser-actions.js

```diff
@@ -13,10 +13,12 @@ const updateBrowserAction = (extensionId, details) => {
 
 const getBrowserAction = (extensionId, details) => {
```

Comment:
just a nit that it would be better to provide a default value for details instead of passing in `{}`